src/conversations/Conversation.ts

```diff
@@ -313,6 +313,10 @@ export class ConversationV1 {
     return decoded
   }
 
+  streamEphemeral(): null {
+    return null
```

Comment:
With how the Conversation Union type works, this is going to make usage awkward. The caller doesn't know whether they have a V1 or V2 conversation without an explicit type check. So, calling `streamEphemeral` could either give you a Promise of a stream or null. Might make more sense to keep the types consistent and just throw an error. Or to figure out how we make ephemeral topics work for V1 conversations.